### ISSUE TITLE
Midterm Peer Review -- ecl93

### ISSUE THREAD

**Issue by elandgre** [tanmaybansal01/orie4741-f19#6]
This project aims to predict locations which are likely to have mass shooting in the future, with the goal of explaining the causes for those shootings. The paper draws from 5 different datasets, focusing on strictness of gun laws, statistics on mass shootings, and other data on the zip codes and demographic breakdown of a region. The project aims to show that lax gun laws are a predictor for mass shootings, and that zip codes also correlate with mass shootings.

Things I liked:
The project attempts to tackle and interesting, important problem
The authors have many ideas for how to combat overfitting, which is necessary given the size of the dataset
The dataset seems well collected, and captures a lot of relevant information

Areas for work:
I would like to see precisely how the score which aggregates the strictness of each states gun laws was computed, with rational for the choice

I would potentially like to see the score for gun laws broken out in to a feature vector corresponding to different aspects of the laws, such as severity of punishment and restrictions placed on gun owners/purchases. I think this could give more explanatory power to the model. For instance, it may be the case that severe punishments are not a strong deterrent for mass shootings, but the other features are, so this could provide interesting insights.

I would like to know more about how accuracy was measured in this project and how the data was split for training and testing. I feel that 60-70% accuracy of predicting mass shootings is hard to interpret without knowing the output space of the problem (i.e. is it classification or predicting probabilities, MSE or L1 ect.)
